tools/project_manager/external_catalog_search.php

```diff
@@ -108,16 +194,18 @@ function do_search_and_show_hits()
     $id = yaz_connect($external_catalog_locator, ["charset" => "UTF-8"]);
     yaz_syntax($id, "usmarc");
     yaz_element($id, "F");
-    yaz_search($id, "rpn", trim($fullquery));
+    yaz_search($id, "rpn", trim(str_replace("\n", " ", $fullquery)));
     $extra_options = ["timeout" => 60];
     yaz_wait($extra_options);
     $errorMsg = yaz_error($id);
 
+    echo "<details style='border: 1px solid #aaa; border-radius: 4px; padding: 0.5em 0.5em 0'>\n";
+    echo "<summary style='font-weight: bold; margin: -0.5em -0.5em 0; padding: 0.5em;'>Raw query</summary>\n";
+    echo "<pre style='padding-left: 4em'>$fullquery</pre>\n";
+    echo "</details>\n";
+
     if (!empty($errorMsg)) {
-        echo "<p class='error'>";
-        echo _("The following error has occurred:");
-        echo " $errorMsg";
-        echo "</p>";
+        echo "<p class='error'>", _("The following error has occurred:"), " $errorMsg", "</p>";
```

Comment:
We should `html_safe($errorMsg)` because while it's probably OK we shouldn't risk it as it comes from an external source. I ack that this was in the original.